package.json

```diff
@@ -90,6 +90,7 @@
     "lint": "npm run eslint && npm run stylelint",
     "eslint": "eslint -f codeframe \"./**/*.{js,jsx}\" ",
     "stylelint": "stylelint \"./**/*.scss\" ",
-    "test": "scripts/run-tests"
+    "test": "scripts/run-tests",
+    "start": "cd packages/components && npm run start"
```

Comment:
let's not use a `start` script in the root package, every subpackage should do that on it's own